tests/fixtures/help/generate.txt

```diff
@@ -1,21 +1,44 @@
+Requested ember-cli commands:
```

Comment:
This diff seems odd.

Reply:
yea, but tests pass, so either:
- this is unused
- tests don't care about this change
- it's been broken?
- a mystery!